client/src/app/containers/Home/ActiveGamesTable.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import {
+    IconButton,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+} from '@material-ui/core';
+import RefreshIcon from '@material-ui/icons/Refresh';
+import { useHistory } from 'react-router-dom';
+
+import { useAuth } from '../../../auth/useAuth';
+import { useGame } from '../Game/useGame';
+import ActiveGames from './ActiveGames';
+import { useActiveGames } from './useActiveGames';
+
+const ActiveGamesTable = () => {
+    const { games, refreshGames } = useActiveGames();
+    const { getGame } = useGame();
```

Comment:
:mag: in the context of the active games table, `getGame` is acting more as `setActiveGame`... :monocle_face:

Reply:
True! Rename incoming. (and this is relevant to a comment above too)